.github/actions/terraform/apply.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Call by action `entrypoint`
+
+SCRIPT_PATH="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
```

Comment:
You can set `WORKDIR /terraform` in the Dockerfile

Reply:
https://docs.github.com/en/actions/creating-actions/dockerfile-support-for-github-actions#workdir

> GitHub sets the working directory path in the GITHUB_WORKSPACE environment variable. It's recommended to not use the WORKDIR instruction in your Dockerfile.